src/odemis/gui/comp/overlay/view.py

```diff
@@ -73,7 +73,7 @@ def _draw_vertical_line(self, ctx, center, size, colour):
         top = center[1] - size
         bottom = center[1] + size
 
-        ctx.set_line_width(1)
+        ctx.set_line_width(2)
```

Comment:
Do you intended to increase the line width of all cross_hairs on all systems? It never annoyed me that the crosshair was to small.
If it is mainly needed for fast-em this could also be an attribute on the crosshair class which can be set by a setter maybe?

Reply:
> Note, this will make the crosshair thicker for all systems. @pieleric is this ok?

Yes, that is with the current implementation the case. It is an open question if this would be useful also on other systems. We got this feedback from user/app and therefore changed it. It is mainly an issue if there is already data acquired and the crosshair is overlaid on the data. There, the visibility is less good. This case would apply for all systems.

Reply:
Sorry I just saw that you included this in a message when you opened this PR. I only looked at the code but skipped that part.

Reply:
No worries. Do you have it in Cryo anywhere that you need to move the stage while already having an image where the crosshair is there as overlay? For CL I don't think we have that use-case.